docs/components/sidecar.md

```diff
@@ -13,7 +13,7 @@ In details:
 * It implements Thanos' Store API on top of Prometheus' remote-read API. This allows [Queriers](./query.md) to treat Prometheus servers as yet another source of time series data without directly talking to its APIs.
 * Optionally, the sidecar uploads TSDB blocks to an object storage bucket as Prometheus produces them every 2 hours. This allows Prometheus servers to be run with relatively low retention while their historic data is made durable and queryable via object storage.
 
-  NOTE: This still does NOT mean that Prometheus can be fully stateless, because if it crashes and restarts you will lose ~2 hours of metrics, so persistent disk for Prometheus is highly recommended. The closest to stateless you can get is using remote write (which Thanos experimentally supports, see [this](../proposals/201812_thanos-remote-receive.md). Remote write has other risks and consequences, and still if crashed you loose in positive case seconds of metrics data, so persistent disk is recommended in all cases.
+  NOTE: This still does NOT mean that Prometheus can be fully stateless, because if it crashes and restarts you will lose ~2 hours of metrics, so persistent disk for Prometheus is highly recommended. The closest to stateless you can get is using remote write (which Thanos experimentally supports, see [Receiver](./receive.md). Remote write has other risks and consequences, and still if crashed you loose in positive case seconds of metrics data, so persistent disk is recommended in all cases.
```

Comment:
```suggestion
  NOTE: This still does NOT mean that Prometheus can be fully stateless, because if it crashes and restarts you will lose ~2 hours of metrics, so persistent disk for Prometheus is highly recommended. The closest to stateless you can get is using remote write (which Thanos supports, see [Receiver](./receive.md). Remote write has other risks and consequences, and still if crashed you loose in positive case seconds of metrics data, so persistent disk is recommended in all cases.
```

is it still experimental?

Reply:
That's a good question actually. Maybe by adding these docs we could say it's not experimental anymore. There's a minor issue that has raised by @brancz https://github.com/thanos-io/thanos/issues/1093#issuecomment-630349045

If we think this could be handled incrementally, I'm happy to remove `experimental` mentions, in this PR. What do you think @thanos-io/thanos-maintainers ?

Reply:
I think for now it's still "experimental" as we want to add one important feature. However we put that for discussion on next Prom Community Meeting cc @brancz

Reply:
I think at least that was my impression, let's ask Frederic. To me it's production ready and I think we can make sure feature we add is compatible

Reply:
I've already taken the initiative and removed experimental mentions.